api/src/main/java/org/openmrs/module/muzimafingerPrint/PatientJsonParser.java

```diff
@@ -97,7 +98,8 @@ public Patient CreatePatient(String patientData) throws JSONException, ParseExce
             patient.setGender((jsonObject.getString("sex")));
 
             String birthDate = jsonObject.getString("birth_date");
-            SimpleDateFormat formatter = new SimpleDateFormat("yyyy-mm-dd");
+            SimpleDateFormat formatter = new SimpleDateFormat("MM/dd/yyyy");
```

Comment:
I think you can use Context.getDateFormat to get default formatter used globally by the current installation of openmrs. Unless you have specific structure that you want to use?